pages/common/josm.md

```diff
@@ -0,0 +1,32 @@
+# josm
+
+> Extensible OpenStreetMap editor for Java 8+.
+> More information: <https://josm.openstreetmap.de/>.
+
+- Launch JOSM:
```

Comment:
Should this be `start` or `launch`?

Reply:
I don't see a difference.

Reply:
Non-native english speakers are more likely to know the word start rather than launch probably

Reply:
I'm not so sure about that. Any other opinions?

Reply:
We use "launch" in a variety of places, and I'm not certain about intentionally simplifying and restricting our vocabulary to a subset of words just because some people might not know them as well.  There are many complex sentence structures in English—being a rather _unlogical_ language—which are likely to elude those who aren't profficient in it; however, there's no need to use them because that would be _useless_ complexity.

w.r.t. to a choice of words though, I don't think that using more _elaborate_ wording is a problem, because any novice could look it up and learn that _launch_ is a **verb**, and **opens an application**.

Makes sense?

Reply:
Just dropping my opinion!

Personally, I'm not against the idea of keeping the English versions accessible to non-native speakers.
The [curb cut effect](https://en.wikipedia.org/wiki/Curb_cut_effect) applies here by being more friendly for:
* Dyslexic users
* Younger demographics
* The tech industry in general, which is dominated by the English language no matter a countries national language

I wouldn't downgrade a sentence for accommodating non-native speakers, though. Launch is a better word (subjective) and basic enough that if someone doesn't know it, they should really learn it.

As a native English speaker, I'd prefer "launch" rather than "start", but in the end it also doesn't make too much difference.

Reply:
I agree, launch definitely is the better word compared to start. Just a suggestion, been discussed and you can resolve if you please

Reply:
Thank you for dropping in @SethFalco! :)